index.html

```diff
@@ -81,11 +81,12 @@ <h1><a rel="home" title="Home" href="index.html" class="floe-header-logo">floe
                         </h1>
                     </header>
                 </div>
-                <div class="columns small-12 medium-3 large-3 align-self-bottom">
+                <div class="columns small-12 medium-12 large-5 align-self-bottom">
                     <nav class="floe-nav">
                         <ul>
-                            <li><a href="news/index.html">News</a></li>
-                            <li><a href="resources.html">Resources</a></li>
+                            <li><a href="./news/index.html">News</a></li>
+                            <li><a href="./resources.html">Resources</a></li>
+                            <li><a href="./ui-options.html">Learner UI Options</a></li>
```

Comment:
Personally I find this confusing as we use the name interchangeably depending on the context. 

I think we should just ask JT, VR, and JM what they think. We should just decide on one name and go with it.

Reply:
Sure, let's check with others to clear things up.

Reply:
I think it's reasonable to just use the "UI Options" name. I think most people know it by that name, it's consistent to the name of the browser extension, and I think the motivations for referring to it as Learner Options are largely historical. I agree with @jhung, it will be less confusing for everyone.